integration/flink/src/main/java/io/openlineage/flink/agent/client/OpenLineageFlinkClient.java

```diff
@@ -0,0 +1,47 @@
+package io.openlineage.flink.agent.client;
+
+import io.openlineage.client.Clients;
+import io.openlineage.client.OpenLineage;
+import io.openlineage.client.OpenLineageClient;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.Properties;
+
+public class OpenLineageFlinkClient {
```

Comment:
EventEmitter? Like in `spark`.

`OpenLineageFlinkClient` suggests that it's OpenLineage's client for Flink.